modules/element.py

```diff
@@ -119,7 +132,8 @@ def __eq__(self, other):
         return all((
             self.isotope == other.isotope,
             self.symbol == other.symbol,
-            self.amount == other.amount
+            self.amount == other.amount,
+            self.sequence == other.sequence
         ))
 
     def __lt__(self, other):
```

Comment:
I think sequence should affect order (if symbol and isotope are equal). The `return str(self) < str(other)` accounts for this somewhat, but something more may be needed.

Reply:
So what do you suggest? If possible, give an example.

Reply:
Similar to `self.isotope`:

```suggestion
    def __lt__(self, other):
    # ...
    
    # Elements that have no sequence number come before elements that do
if self.sequence is None and other.sequence is not None:
    return True

if self.sequence is not None and other.sequence is None:
    return False

return str(self) < str(other)
```